decimal64math.go

```diff
@@ -5,8 +5,13 @@ func (d Decimal64) Abs() Decimal64 {
 	return Decimal64{^neg64 & uint64(d.bits)}
 }
 
-// Add computes d + e
+// Add computes d + e with default rounding set to half_up
 func (d Decimal64) Add(e Decimal64) Decimal64 {
+	return defaultContext.Add(d, e)
+}
+
+// Add computes d + e
+func (context DecContext) Add(d, e Decimal64) Decimal64 {
```

Comment:
Keep `DecContext` methods together, maybe even seperate `Decimal64` and `DecContext` into separate files.

Reply:
now Decimal64 methods come first, context64 methods come later